### DISCUSSION TITLE
Quotes Module

### DISCUSSION THREAD

**Discussion by groiso** [YetiForceCompany/YetiForceCRM#15848] [General]
Hi,

Within Quotes Module, there are columns referring to products/services.

I would like to know how I can delete and insert a column.

How could I put a custom field in the Product Module, in the Quotes column.

Thanks

**Comment by mariuszkrzaczkowski** [comment]
https://gitdeveloper.yetiforce.com/index.php?module=LayoutEditor&parent=Settings&view=Index&sourceModule=SQuotes
![image](https://user-images.githubusercontent.com/10198654/161402860-ad572260-f218-4061-9bdc-5987eee84d08.png)

**Reply by groiso** [reply]
Thank you very much, it worked.

**Reply by groiso** [reply]
Hi, @mariuszkrzaczkowski 

If I create a field in the Product Module, how can I make this field appear between these columns?

**Reply by mariuszkrzaczkowski** [reply]
these are independent data

**Reply by groiso** [reply]
Hi, @mariuszkrzaczkowski 

Yes, they are independent data.

Example in the Product Module I created a field called "Commissions" 10%.

How could this field "Commissions" with the same 10% appear in the column of the Quotes Module, with the same 10% value created in the Product?

**Reply by mariuszkrzaczkowski** [reply]
there is a mechanism for this, it has already been described on github, look for it

**Comment by groiso** [comment]
Hi, @mariuszkrzaczkowski 

Search and browse the 26 pages of Discussions and I didn't find it. Could you tell me exactly the # of the question?
Thanks

**Comment by rskrzypczak** [comment]
You need some programming skills.
Example: https://github.com/YetiForceCompany/YetiForceCRM/issues/12945, https://github.com/YetiForceCompany/YetiForceCRM/discussions/15701, https://github.com/YetiForceCompany/YetiForceCRM/discussions/14800, etc.

**Comment by groiso** [comment]
Thanks
